modules/migration-mtc-workflow.adoc

```diff
@@ -56,10 +56,12 @@ image::migration-PV-move.png[]
 
 . Run the migration plan, with one of the following options:
 
-* *Stage* (optional) copies data to the target cluster without stopping the application.
+* *Stage* copies data to the target cluster without stopping the application.
 +
-Staging can be run multiple times so that most of the data is copied to the target before migration. This minimizes the duration of the migration and the application downtime.
+A stage migration can be run multiple times so that most of the data is copied to the target before migration. This reduces the duration of the cutover migration.
 
-* *Migrate* stops the application on the source cluster and recreates its resources on the target cluster. Optionally, you can migrate the workload without stopping the application.
+* *Cutover* stops the application on the source cluster and moves the resources to the target cluster.
++
+Optional: You can clear the *Halt transactions on the source cluster during migration* check box.
```

Comment:
Per the ISG, "checkbox" is one word.

Reply:
Not any more. Check the lastest IBM guide. It is now "check box".

Reply:
Interesting. IBM has both. They have "checkbox" in the Usage section and "check box" everywhere else. I'll go with the one-word version because that's what we have been doing up to now.